### ISSUE TITLE
[go] /usr/local/work isn't FHS compliant

### ISSUE THREAD

**Issue by saltedcoffii** [chromebrew/chromebrew#5594]
Our go package symlinks ~/go to /usr/local/work/go. This isn't anywhere close to being FHS compliant or even standard. We should probably find another location for it.

**Comment by satmandu** [comment]
Do the env variable files in the go package even make sense?

What is the `go-run` section even doing?

```

    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
    @gorun = <<~EOF
      # Uncomment the line starting with "export" below to use `go run`
      # Don't forget to uncomment it when you're done
    EOF
    IO.write("#{CREW_DEST_PREFIX}/etc/env.d/go-run", @gorun)
    @godev = <<~EOF
      # Go development configuration
      if [ ! -e #{CREW_PREFIX}/work/go ]; then
        mkdir -vp #{CREW_PREFIX}/work/go
      fi
      if [ ! -e #{HOME}/go ]; then
        ln -sv #{CREW_PREFIX}/work/go #{HOME}/go
      fi
      export PATH="$PATH:$HOME/go/bin"
    EOF
    IO.write("#{CREW_DEST_PREFIX}/etc/env.d/go-dev", @godev)
  end

  def self.postinstall
    puts
    puts "Edit #{CREW_PREFIX}/etc/env.d/go-run to be able to use go-run".lightblue
    puts
  end
```

**Comment by Zopolis4** [comment]
Fixed by #7741.
